plugin.video.twitch/resources/lib/twitch_addon/routes/install_ircchat.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+from ..addon.common import kodi
+
+
+def route():
+    if kodi.get_kodi_version().major > 16:
```

Comment:
Is this run automatically? Can you please provide a yesno dialog so the user knows what is happening on the background?

Reply:
This is called by a settings action, and both the runscript and installaddon have builtin yes/no Kodi builtin install confirmation.